lib/cisco_node_utils/bgp_af.rb

```diff
@@ -66,6 +67,26 @@ def destroy
       config_set('bgp', 'address_family', @set_args)
     end
 
+    def feature_nv_overlay_enabled?
+      config_get('feature', 'nv_overlay')
```

Comment:
I like the idea of moving these into a feature.yaml but I don't like all the duplication in the different providers. 
I'd prefer to have a feature provider where these can all be kept centrally and then just call `Feature.nv_overlay_enabled?` instead. Is that doable?